### PR TITLE
feat: People Search

### PR DESCRIPTION
**Pull request by vinnyjth**
### What does this PR do, or why is it needed?

People search! Using the basis of Jimmy and Luke's excellent work. 

A few notes about this PR: 
- Pagination is un-implemented on the front end. I think that's the right choice for now. 
- Still some UI cleanup to be done on the search modal. We can tackle that in another PR. 

### How do I test this PR?

Visit the connect tab while linked and hit that shiny "Find People to People" button. Probs should fix my typo 🙃

### CODE REVIEW

File: packages/apollos-ui-connected/src/ConnectScreenConnected/index.js [hunk 1]
```diff
@@ -41,27 +42,40 @@ const ConnectScreenConnected = (props) => {
...
+              />
+            }
+          >
+            <UserAvatarHeaderConnected />
+            {ActionBar && <ActionBar />}
+            <RequestedFollowListConnected refetchRef={refetchRef} />
+            <SuggestedFollowListConnected
+              refetchRef={refetchRef}
+              onPressFollowListButton={() => setModalOpen(true)}
+              followListButtonTitle={'Find People to People'}
```
Comment (vinnyjth) [inline packages/apollos-ui-connected/src/ConnectScreenConnected/index.js:65]:
```suggestion
              followListButtonTitle={'Find People to Follow'}
```

**Comment by jimmyhogoboom** [comment]
@vinnyjth I'm not seeing the button, getting `GraphQLError: Cannot query field "currentUserFollowing" on type "Person".` Is there a templates branch I need?

**Comment by vinnyjth** [comment]
image

@jimmyhogoboom I just tested on a different machine than what I wrote the PR on ^ Do you have the latest `master` pulled down on `templates`?

**Comment by jimmyhogoboom** [comment]
🤦‍♂️

**Comment by jimmyhogoboom** [comment]
We might need a special case to handle the button state for suggested people, since they accept immediately.


https://user-images.githubusercontent.com/61669826/111827956-8f08a300-88b8-11eb-818f-52eff1420286.mp4

**Comment by vinnyjth** [comment]
> We might need a special case to handle the button state for suggested people, since they accept immediately.

Ohh, that's a really good catch! I'll probs tackle that in another PR since it's an issue with the FollowListConnected :)

**Comment by jimmyhogoboom** [comment]
Ok now for some reason my search button is gone.

Simulator Screen Shot - iPhone 11 - 2021-03-19 at 14 01 57

**Comment by vinnyjth** [comment]
@jimmyhogoboom here's the fix for the follow list fyi: https://github.com/ApollosProject/apollos-apps/pull/1816

**Comment by vinnyjth** [comment]
@jimmyhogoboom ah good catch. that's because the SuggestedFollowLIst hides if there are no suggestions left, taking the button with it. I'll fix that here!

**Comment by vinnyjth** [comment]
@jimmyhogoboom Pushed up a fix for the button goin away :)

**Review by jimmyhogoboom — APPROVED** [review]
Lookin good!

**Pull request merged**